content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+﻿---
+layout: blog
+title: Get started with Kubernetes (using Python)
+date:  
+---
+
+**Author**: Jason Haley (Independent Consultant)
+
+So, you know you want to run your application in Kubernetes but don’t know where to start. Or maybe you’re getting started but still don’t know what you don’t know. In this blog you’ll walk through how to containerize an application and get it running in Kubernetes.
+
+This walk-through assumes you are a developer or at least comfortable with the command line (preferably bash shell).
+
+## What we’ll do
+1. Get the code and run the application locally
+2. Create an image and run the application in Docker
+3. Create a deployment and run the application in Kubernetes
+
+## Prerequisites
+* [Docker Desktop with Kubernetes](https://www.docker.com/products/kubernetes) installed (see end of blog for enabling Kubernetes)
+* [Python 3.7](https://www.python.org/) installed
+* [Git](https://git-scm.com/downloads) installed
+
+## Containerizing an application
+In this section you’ll take some source code, verify it runs locally, and then create a Docker image of the application. The sample application used is a very simple Flask web application; if you want to test it locally, you’ll need Python installed. Otherwise, you can skip to the "Create a Dockerfile" section.
+
+### Get the application code
+
+Use git to clone the repository to your local machine:
+```
+git clone https://github.com/JasonHaley/hello-python.git
+```
+
+Change to the app directory:
+```
+cd hello-python/app
+```
+
+There are only two files in this directory. If you look at the main.py file, you’ll see the application prints out a hello message. You can learn more about Flask on the [Flask website](http://flask.pocoo.org/).
+
+```
```

Comment:
Hi. If you wanted to have this highlighted as Python you could do that (append `python` to this line). What do you think?

Reply:
Not a bad idea. It would make sense that someone may want to also learn more about Python, though originally I didn't really want to highlight a specific language. My original title didn't have it in there either but I got some feedback that it might make sense to put it in the title.